firmware/cartridges/cartridge.c

```diff
@@ -30,6 +30,7 @@
 #include "super_snapshot_5.c"
 #include "easyflash.c"
 #include "comal80.c"
+#include "zaxxon.c"
```

Comment:
I tend to order the includes in the same order as the CRT number. That makes it a bit easier (for me at least)

Reply:
Absolutely. That is actually what I expected but then I got confused when trying to validate this assumption.